c3d/README.rst

```diff
@@ -0,0 +1,54 @@
+Code style
```

Comment:
We shouldn't have a second readme inside the code folder.

Is this related to the documentation tool you are using?